src/Presenters/ApiPresenter.php

```diff
@@ -240,4 +243,37 @@ private function log($apiIdentifier, $authorization, $response)
             'userAgent' => $userAgent,
         ]), HermesMessage::PRIORITY_LOW);
     }
+
+    private function hasPathLogEnabled(string $path): bool
+    {
+        $apiLogPathsConfig = Strings::trim($this->applicationConfig->get('enabled_api_log_paths'));
+        $logPaths = array_filter(array_map(function ($value) {
+            return trim($value);
+        }, preg_split("/\r\n|\r|\n/", $apiLogPathsConfig)));
+
+        if (count($logPaths) === 0) {
+            return true;
+        }
+
+        $pathParts = explode('/', trim($path, '/'));
+        if (count($pathParts) !== self::API_PATH_PARTS) {
```

Comment:
- Is it necessary to do this kind of splitting? Maybe I'm oversimplifying but wouldn't `strpos()` be enough?
- Btw. if we use DI and `setup` method, maybe there is a way how to initialize this into the array, so you could only call `isset($whitelist[$path])`.

Reply:
- exploded parts was later used for "simple" version of "* matching" for definitions like /v1/users/* etc.